_posts/2018-7-18-EIA-coal.md

```diff
@@ -0,0 +1,141 @@
+
+The plot below gives an overview of how the coal produced each year differs in the amount of sulfur content.  To construct the plot, we did the following:
+
+* Per year, sort all coal shipments from those with the highest percentage of sulfur to those with the lowest.
+* Using this order, calculate the cumulative quantity.  The last value is the total amount of coal produced in the US in that year.
+
+A useful feature of this type of plot is that the area under the curve is the total amount of sulfur contained in coal shipments that year.  Instead of reducing the yearly amount of sulfur to a single number, this plot shows how it is distributed based on the properties of the coal shipped.
+
+![Coal Sulfur]({{ site.baseurl }}public/images/YearlyCoalSulfurQuantity.gif)
+<sup>_Profile of coal sulfur content._</sup>
+
+For reference, we use 2008 (in blue) as a baseline since that is the first year in the EIA data.  As the animation progresses, we can see that total coal production peaks in 2010, before steadily decreasing to levels below 2008.  By examining the differences between the two curves, we can see where increases and decreases sulfur from different types of coal have occurred.
+
+For example, on the left side of the plot, the gray areas show increased amount of sulfur from coal which is high in sulfur.  Later in the animation, we see light blue areas, representing decreased amounts of sulfur from low-sulfur coal (and less coal production overall).  By subtracting the size of the light blue areas from the gray areas, we can calculate the overall change in sulfur, relative to 2008.
+
+As described further below in this post, electricity generation from coal has decreased, although it has been observed that [SO<sub>2</sub> emissions have fallen quicker than the decrease in generation](https://www.eia.gov/todayinenergy/detail.php?id=29812){:target="_blank"}, in part due to more stringent desulfurization requirements between 2015 and 2016.  The increased production of high-sulfur coal shown in the plot suggests an economic tradeoff, which would be interesting to explore with a more detailed analysis.  For example, while [low-sulfur coal commands a higher price](https://www.eia.gov/energyexplained/index.php?page=coal_prices){:target="_blank"}, one could also choose high-sulfur coal, but then be faced with the costs of operating the required desulfurization technologies.
```

Comment:
--> "As discussed further below, electricity generation from coal has decreased, with [SO<sub>2</sub> emissions falling faster than the decrease in generation](https://www.eia.gov/todayinenergy/detail.php?id=29812){:target="_blank"}, in part due to more stringent desulfurization requirements between 2015 and 2016. "

Reply:
--> "For example, while [low-sulfur coal commands a higher price](https://www.eia.gov/energyexplained/index.php?page=coal_prices){:target="_blank"}, it is possible for a generator to choose high-sulfur coal, but then be faced with the costs of operating the required desulfurization technologies."